Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -7581,13 +7582,12 @@
     "Context": {
       "ReferenceDateTime": "2018-07-17T13:00:00"
     },
-    "Comment": "RELPERIOD. Refine support for relative periods.",
-    "NotSupported": "dotnet, javascript, python, java",
+    "NotSupported": "javascript, python, java",
     "Results": [
       {
-        "Text": "el año anterior",
-        "Start": 20,
-        "End": 34,
+        "Text": "año",
```

Comment:
Isn't this incorrect? I'd rather it recognize "the previous year" in such sentences.

Reply:
We can keep this behaviour for now. But please open an issue for this case and tag it with the 'open question' label.

Reply:
The problem is that this pattern is ambiguous, it could mean "the previous year", but in this context it means "the year before the revolution".
So I conformed it with the English behaviour where for the case "The story happened the year before independence." only "the year" is extracted.
Maybe I could modify the behaviour to take into account "anterior" when it is not followed by prepositions.

Reply:
anyway, yes I will open the issue